mkdocs.yml

```diff
@@ -0,0 +1,31 @@
+site_name: OWASP Cheat Sheet Series
+site_description: Website with the collection of all the cheat sheets of the project.
+docs_dir: cheatsheets/
+plugins:
+    - search:
+        prebuild_index: true
+#theme:
```

Comment:
Can we add google analytics here ? 
`google_analytics: ['UA-4531126-15', 'auto']`

This will solve #386

Reply:
can be done at the top level, not in the plugin part, but yes in my opinion.

Reply:
Can you please inject it as well? That'd be a great help.